### DISCUSSION TITLE
where and how to contribute?

### DISCUSSION THREAD

**Discussion by DJCrashdummy** [privacytoolsIO/privacy-tools#18] [General Privacy Forum]
i'm sorry if this fits better at https://github.com/privacytoolsIO/privacy-tools/discussions/1... but i wasn't sure.

---

@privacytoolsIO please state somewhere clearly visible (e.g. at the website or at least at the readme in the repo) how to contribute to privacytools.io, if it is still possible and wanted.

as there is no usual way (via issues) and merge request are also impossible as it seems that there is nothing of the site or any other resources, i'm pretty sure a significant number of people didn't contribute at all although the wanted to. i was also about to give up until i found the discussions "by accident" at the last moment.

please keep in mind: as privacytools.io gets more and more outdated, more and more people will assume that this project is dead or at least abandoned and hence won't even try to contribute. :frowning_face:

**Comment by SeryiBaran** [comment]
++, where to find repo with content?

**Reply by I-I-IT** [reply]
Privacy Tools is now just [affiliation](https://www.privacytools.io/help/article/4-AdGuard-Affiliate-Partner-Program-Review-50-Commission) money crap. 

Please look at Privacy Guides(privacyguides dot org), a community-based sucessor.

**Reply by BigBlueHat** [reply]
First click I made was on [Internxt Photos](https://www.privacytools.io/go/internxt.com) which opens a tab...redirects several times (...check it in `curl`) and then closes the tab.

Nothing "private" about that... 😠
